dist/js/sth-select.js

```diff
@@ -195,15 +195,16 @@
 		/**
    * Renders all elements in the list of options.
    */
-		function _renderList() {
+		function _renderList(caseSensitive) {
 			_clear();
 
 			var rerenderOnEachItem = false;
 			var $listItems = $([]);
-			var textFilter = _$filter.val().toLowerCase();
+			var textFilter = formatText(caseSensitive, _$filter.val());
 
-			_items.map(function (item) {
-				if (item.text.toLowerCase().indexOf(textFilter) != -1) {
+			_items.forEach(function (item) {
```

Comment:
We should use `forEach` here instead of map because `map` returns new array and we dont need it here

Reply:
Great! You're correct. 👍